commit 7e90db0ddcc2b4bb3139ae2852b342e30250618d
Author: Jeremy Huddleston <jeremyhu@apple.com>
Date:   Thu Oct 20 22:54:08 2011 -0700

    apple: Silence some debug spew
    
    Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
    (cherry picked from commit 098ecfad83a63bd8eb04c37f268c18d8744dff2c)

diff --git a/src/glx/apple/appledri.c b/src/glx/apple/appledri.c
index 46c84f3..e3f9b84 100644
--- a/src/glx/apple/appledri.c
+++ b/src/glx/apple/appledri.c
@@ -332,12 +332,12 @@ XAppleDRICreateSharedBuffer(Display * dpy, int screen, Drawable drawable,
       return False;
    }
 
-   printf("rep.stringLength %d\n", (int) rep.stringLength);
+   /* printf("rep.stringLength %d\n", (int) rep.stringLength); */
 
    if (rep.stringLength > 0 && rep.stringLength <= pathlen) {
       _XReadPad(dpy, path, rep.stringLength);
 
-      printf("path: %s\n", path);
+      /* printf("path: %s\n", path); */
 
       *width = rep.width;
       *height = rep.height;
@@ -404,7 +404,7 @@ XAppleDRICreatePixmap(Display * dpy, int screen, Drawable drawable,
    if (rep.stringLength > 0 && rep.stringLength <= bufnamesize) {
       _XReadPad(dpy, bufname, rep.stringLength);
 
-      printf("path: %s\n", bufname);
+      /* printf("path: %s\n", bufname); */
 
       *width = rep.width;
       *height = rep.height;
